scripts/pagerduty_notify.rb

```diff
@@ -82,6 +86,16 @@ def check_arguments(arglist)
 ap output_data if(ENV['debug'])
 #JSON.dump(output_data)
 
+disable_flag=false
+
+if ENV['disable_communications']
+  disable_flag=$store.substitute_string(ENV['disable_communications'])
+  if disable_flag.downcase=='yes' or disable_flag.downcase=='true' or disable_flag.downcase=='disable'
```

Comment:
> a couple of permuations 

This will definitely be confusing to any other developers looking at this - it's exactly what a boolean was invented for.